pkg/api/aim/runs.go

```diff
@@ -1006,6 +1061,12 @@ func SearchAlignedMetrics(c *fiber.Ctx) error {
 				}
 				values = append(values, v)
 				iters = append(iters, float64(metric.Iter))
+				if metric.Context != nil {
```

Comment:
if metric.Context were a `map[string]any` instead of `datatypes.Json`, could we avoid these unmarshalling steps? might be worth considering